### PR TITLE
Lrdocs 6139 command line fragments

### PR DESCRIPTION
**Pull request by stephenkostas**
@p2kmgcl - for review

### CODE REVIEW

File: develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/01-setting-up-the-fragments-generator.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+3. Yeoman 2+ (link)
+
+## Installing Yeoman and the Fragment Generator
+
+If you don't already have node.js and NPM installed, [follow these instructions](link). Once you have them installed, installing the Fragment 
+Generator only takes a couple steps.
+
+1.  From the command line, run `npm install -g yo` to install Yeoman.
+
+2.  Run, `npm-install -g generator liferay-fragments` to install the Fragment
```
Comment (p2kmgcl) [inline develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/01-setting-up-the-fragments-generator.markdown]: Typo `npm install`, not `npm-install`

File: develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Using the Fragment Generator
+
+After you setup the generator and create your first project by running
+`yo liferay-fragment` you will have the basic structure created, but there's
```
Comment (p2kmgcl) [inline develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown]: `yo liferay-fragments`, not `yo liferay-fragment`

File: develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+still more that the generator can help you with.
```
Comment (p2kmgcl) [inline develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown]: Maybe we could mention here that all existing task are defined inside `package.json > scripts`. They're just yeoman generator aliases

File: develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/01-setting-up-the-fragments-generator.markdown (hunk 1 above)
Comment (stephenkostas) [reply develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/01-setting-up-the-fragments-generator.markdown]: Fixed missing dash.

File: develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown (hunk 2 above)
Comment (stephenkostas) [reply develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown]: Corrected command to "yo liferay-fragments"

File: develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown (hunk 3 above)
Comment (stephenkostas) [reply develop/tutorials/articles/230-web-experience-management/xx-liferay-fragments-generator/02-using-the-fragment-generator.markdown]: Added info that "You can see all of the available tasks inside the `scripts` section in `package.json`."

**Comment by p2kmgcl** [comment]
:+1:

**Pull request closed without merging**